File: Java/Matrix.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the number of rows: ");
        int rows = scanner.nextInt();

        System.out.print("Enter the number of columns: ");
        int columns = scanner.nextInt();

        // Crear una matriz
        int[][] matrix = new int[rows][columns];

        // Asignar valores a la matriz
        for (int i = 0; i < rows; i++) {
            for (int j = 0; j < columns; j++) {
                System.out.print("Enter the element at position (" + i + ", " + j + "): ");
                matrix[i][j] = scanner.nextInt();
            }
        }

        // Imprimir la matriz
        System.out.println("Matrix:");
        for (int i = 0; i < rows; i++) {
            for (int j = 0; j < columns; j++) {
                System.out.print(matrix[i][j] + " ");
            }
            System.out.println();
        }
    }
}

File: Kotlin/Matrix.kt
import java.util.*

fun main() {
    val scanner = Scanner(System.`in`)
    
    print("Enter the number of rows: ")
    val rows = scanner.nextInt()
    
    print("Enter the number of columns: ")
    val columns = scanner.nextInt()
    
    // Crear una matriz
    val matrix = Array(rows) { IntArray(columns) }
    
    // Asignar valores a la matriz
    for (i in 0 until rows) {
        for (j in 0 until columns) {
            print("Enter the element at position ($i, $j): ")
            matrix[i][j] = scanner.nextInt()
        }
    }
    
    // Imprimir la matriz
    println("Matrix:")
    for (i in 0 until rows) {
        for (j in 0 until columns) {
            print("${matrix[i][j]} ")
        }
        println()
    }
}
